coral/src/app/features/topics/request/TopicRequest.test.tsx

```diff
@@ -410,4 +439,43 @@ describe("<TopicRequest />", () => {
       });
     });
   });
+
+  describe("AdvancedConfiguration", () => {
+    beforeAll(async () => {
+      mockGetEnvironmentsForTeam({
+        mswInstance: server,
+        response: {
+          data: [createMockEnvironmentDTO({ name: "DEV", id: "1" })],
+        },
+      });
+
+      mockGetTopicAdvanvedConfigOptions({
+        mswInstance: server,
+        response: {
+          data: defaultGetTopicAdvanvedConfigOptionsResponse,
+        },
+      });
+
+      customRender(
+        <AquariumContext>
+          <TopicRequest />
+        </AquariumContext>,
+        { queryClient: true }
+      );
+
+      await screen.findByLabelText("Environment");
+      await screen.findByRole("option", { name: "DEV" });
+    });
+    afterAll(() => {
+      cleanup();
+    });
+
+    it("has field which accepts value", async () => {
```

Comment:
Maybe the test description could be a bit more detailed? :D

Reply:
I am not able to come up with anything better. The real tests come, when I implement the form submission. Added tests for the sub heading and  help text though.